demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -625,6 +655,94 @@ static void handleIncomingPublish( MQTTPublishInfo_t * pPublishInfo,
 
 /*-----------------------------------------------------------*/
 
+static void updateSubAckStatus( MQTTPacketInfo_t * pPacketInfo )
+{
+    uint8_t * pPayload = NULL;
+    size_t pSize = 0;
+
+    MQTTStatus_t mqttStatus = MQTT_GetSubAckStatusCodes( pPacketInfo, &pPayload, &pSize );
+
+    /* MQTT_GetSubAckStatusCodes always returns success if called with packet info
+     * from the event callback and non-NULL parameters. */
+    assert( mqttStatus == MQTTSuccess );
+
+    /* Suppress unused variable warning when logging is disabled. */
```

Comment:
This has nothing to do with logging
```suggestion
    /* Suppress unused variable warning when asserts are disabled. */
```

Reply:
Fixed, thanks.